src/plugins/chromecastPlayer/chromecastHelper.js

```diff
@@ -1,229 +1,229 @@
+    function ensureProtocol(url) {
+        if (!url.match(protocolRegExp)) {
+            url = 'http://' + url;
         }
+        return url;
+    }
 
-        // look for links in the text
-        var LinkParser = {
-            parse: function (text) {
-                var links = [];
-                var match;
-
-                // eslint-disable-next-line no-cond-assign
-                while (match = linkRegExp.exec(text)) {
-                    console.debug(match);
-                    var txt = match[0];
-                    var pos = match.index;
-                    var len = txt.length;
-                    var url = ensureProtocol(text);
-                    links.push({ 'pos': pos, 'text': txt, 'len': len, 'url': url });
-                }
-
-                return links;
+    // look for links in the text
+    const LinkParser = {
+        parse: function (text) {
+            const links = [];
+            let match;
+
+            // eslint-disable-next-line no-cond-assign
+            while (match = linkRegExp.exec(text)) {
+                console.debug(match);
+                const txt = match[0];
+                const pos = match.index;
+                const len = txt.length;
+                const url = ensureProtocol(text);
+                links.push({ 'pos': pos, 'text': txt, 'len': len, 'url': url });
             }
 
-        };
+            return links;
+        }
 
-        window.LinkParser = LinkParser;
-    })();
+    };
 
-    var cache = {};
+    window.LinkParser = LinkParser;
+})();
 
-    function isValidIpAddress(address) {
-        var links = LinkParser.parse(address);
+let cache = {};
 
-        return links.length == 1;
-    }
+// TODO: Replace with isIP (https://www.npmjs.com/package/is-ip)
+function isValidIpAddress(address) {
+    const links = LinkParser.parse(address);
 
-    function isLocalIpAddress(address) {
-        address = address.toLowerCase();
+    return links.length == 1;
+}
 
-        if (address.indexOf('127.0.0.1') !== -1) {
-            return true;
-        }
-        if (address.indexOf('localhost') !== -1) {
-            return true;
-        }
+// TODO: Add IPv6 support. Potentially replace with isLocalhost (https://www.npmjs.com/package/is-localhost-ip)
+function isLocalIpAddress(address) {
+    address = address.toLowerCase();
 
-        return false;
+    if (address.includes('127.0.0.1')) {
+        return true;
+    }
+    if (address.includes('localhost')) {
+        return true;
     }
 
-    function getServerAddress(apiClient) {
-        var serverAddress = apiClient.serverAddress();
-
-        if (isValidIpAddress(serverAddress) && !isLocalIpAddress(serverAddress)) {
-            return Promise.resolve(serverAddress);
-        }
+    return false;
+}
 
-        var cachedValue = getCachedValue(serverAddress);
-        if (cachedValue) {
-            return Promise.resolve(cachedValue);
-        }
+function getServerAddress(apiClient) {
```

Comment:
Same as `default`.
```suggestion
export function getServerAddress(apiClient) {
```
tbh, I'm currently not sure if we should export each function/class.

Reply:
It's a decision we'll have to come back in once the Webpack PR is done, imo.

The "export default" route means we import everything, even stuff we don't need (And it ends up in the bundle). It also didn't work for me in some rare cases (Exporting a new anonymous instance of a class, for example).

I've changed a few files to use only "export whatever", especially those anonymous instances of a class.